cooperator/models/subscription_request.py

```diff
@@ -316,7 +318,9 @@ def _compute_subscription_amount(self):
     phone = fields.Char(
         string="Phone", readonly=True, states={"draft": [("readonly", False)]}
     )
-    user_id = fields.Many2one("res.users", string="Responsible", readonly=True)
+    user_id = fields.Many2one(
+        "res.users", string="Responsible", readonly=True, check_company=True
```

Comment:
`res.users` multi-company is contemplated by `ir.rules`, check if this is really required.

Reply:
The ir.rules only govern _access_ rules for users of Odoo. `check_company` is a data consistency measure. It's like an SQL constraint that disallows the company_id of user_id to differ from the company_id of the subscription request.

This shouldn't _happen_ in normal usage of the module, I think, but inconsistent data is Bad and Evil and should be prevented.

Reply:
agreed with @carmenbianca. normal usage should not create problems, but this ensures that inconsistent data cannot be created by code using this module.